### DISCUSSION TITLE
Future integrations with AWS-CDK/other IaC tools?

### DISCUSSION THREAD

**Discussion by eddavies6** [infracost/infracost#1870] [General]
Hello, 

Really enjoying using Infracost for our terraform deployments to give an estimated breakdown for costs! Just wondering if there were any future plans to integrate with other IaC tools such as AWS-CDK , we also use this for some deployments and would be very useful if Infracost supported this. 

Apologies if this has already been asked or if it is completely out of scope!

Best regards

**Comment by alikhajeh1** [comment]
Hi @eddavies6 - great question! We do have plans, there's a community member from Pulumi [working](https://github.com/infracost/infracost/pulls) on Pulumi integration, there are issues for [Terraform CDK](https://github.com/infracost/infracost/issues/1219), [Azure ARM](https://github.com/infracost/infracost/issues/812) and [CloudFormation](https://github.com/infracost/infracost/issues/190). Feel free to create a GH issue for AWS-CDK so the community can subscribe to it for updates.

**Comment by glenngillen** [comment]
Hey @eddavies6, I just discovered this discussion from a few years ago. We now [support CDK](https://www.infracost.io/docs/features/cdk/)!
